File: C++/C++Basic/04.Conditional-Statements-Exercise/01.sumSeconds.cpp
#include <iostream>

using namespace std;

int main(){
    int timeFirst, timeSecond, timeThird;
    cin >> timeFirst >> timeSecond >> timeThird;

    int sumOfTime = timeFirst + timeSecond + timeThird;

    int minutes = sumOfTime / 60;
    int second = sumOfTime % 60;

    if (second < 10){
        cout << minutes << ":0" << second << endl;
    }
    else {
        cout << minutes << ":" << second << endl;
    }

    return 0;
}

File: Python/Basic/04.Conditional-Statements-Exercise/01.sum_seconds.py
time_first = int(input())
time_second = int(input())
time_third = int (input())

total_second = time_first + time_second + time_third
minutes = total_second // 60
seconds = total_second % 60

result = ''

if seconds < 10:
    result = f'{minutes}:0{seconds}'
else:
    result = f'{minutes}:{seconds}'

print(result)